### ISSUE TITLE
Add a non vision related, structured data classification example aka XGBoost, LGBM or Scikitlearn regression model

### ISSUE THREAD

**Issue by vkullu** [NVIDIA/NVFlare#177]
A lot of us practitioners in the health care space, dealing with genomic and EMR data do not use deep learning by default. Gradient boosting methods like XGBoost, LGBM or Scikitlearn helps with majority of the non deep learning use cases. Keras provides simple APIs to get model weights easily which is much trickier in the other methods I explained above. 

Providing a classification example with one of those models will help showcase how to implement those complexities. I can provide sample datasets if needed.

**Comment by holgerroth** [comment]
Thank you for this suggestion. For others interested in this topic, please also see the discussion [here](https://github.com/NVIDIA/NVFlare/discussions/122).

**Comment by holgerroth** [comment]
FYI, there has been some development in this direction by @rongou for XGBoost https://github.com/dmlc/xgboost/tree/master/demo/nvflare

**Comment by vkullu** [comment]
Thank you @holgerroth . A good samaritan shared that branch with us recently and we have delightfully started experimenting with it. Much appreciated!
